### ISSUE TITLE
Connect cli with reporter-earl?

### ISSUE THREAD

**Issue by dmundra** [dequelabs/axe-core-npm#221]
Hi,

Is there a way for the axe CLI tool to generate an EARL formatted JSON report? If there is a recommendation on combining the two packages that would be great.

Thank you
Daniel

**Comment by wonderfulspam** [comment]
I'm having similar issues trying to combine `@axe-core/puppeteer` and the EARL reporter as mentioned in https://github.com/dequelabs/axe-core-npm/issues/88#issuecomment-811232521. Would also be grateful for an example that shows how to integrate the EARL reporter.

**Comment by la55u** [comment]
Did you find a solution on how to make the CLI use the `reporter-earl` package?

**Comment by dmundra** [comment]
Not yet. @mgifford any luck?

**Comment by mgifford** [comment]
I remember that there was an error in the version of axe that I was using, but think that was fixed by an update that @WilcoFiers did. 

I wasn't able to get it in an EARL format. 

It is also possible that EARL isn't as useful a format.  Microsoft has taken the approach of building their own formatting, as has Deque. 

Also worth pointing to https://github.com/microsoft/accessibility-insights-web/issues/4398

**Comment by WilcoFiers** [comment]
@mgifford I would think so. If you have a situation like in [ACT](https://www.w3.org/WAI/standards-guidelines/act/implementations/), where you have to compare results from different testers, it's really the only choice. But it's a very difficult format to understand and work with in my opinion. To get developers to use it in ACT, we had to dumb it down into copy-past'able examples code snippets, and still I hear from tool vendors that EARL is one of the biggest obstacles to contribute to ACT.
